App Crashes when no internet connection on the time of opening the app

**Actual Behaviour**

When no internet connection available while opening the app then the App automatically crashes.

**Expected Behaviour**

If no internet connection available then App must open the NoNetworkActivity. 

**Steps to reproduce it**
The problem occurs on ButterKnife to bind the button.

**LogCat for the issue**

```
05-24 12:14:34.794 24877-24877/com.codingblocks.chatter E/AndroidRuntime: FATAL EXCEPTION: main
    Process: com.codingblocks.chatter, PID: 24877
    java.lang.RuntimeException: Unable to start activity ComponentInfo{com.codingblocks.chatter/com.codingblocks.chatter.NoNetworkActivity}: java.lang.NullPointerException: Attempt to invoke virtual method 'void android.widget.Button.setOnClickListener(android.view.View$OnClickListener)' on a null object reference
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2455)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2515)
        at android.app.ActivityThread.access$1000(ActivityThread.java:154)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1379)
        at android.os.Handler.dispatchMessage(Handler.java:102)
        at android.os.Looper.loop(Looper.java:157)
        at android.app.ActivityThread.main(ActivityThread.java:5571)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:745)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:635)
     Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'void android.widget.Button.setOnClickListener(android.view.View$OnClickListener)' on a null object reference
        at com.codingblocks.chatter.NoNetworkActivity.onCreate(NoNetworkActivity.java:34)
        at android.app.Activity.performCreate(Activity.java:6357)
        at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1108)
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2408)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2515) 
        at android.app.ActivityThread.access$1000(ActivityThread.java:154) 
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1379) 
        at android.os.Handler.dispatchMessage(Handler.java:102) 
        at android.os.Looper.loop(Looper.java:157) 
        at android.app.ActivityThread.main(ActivityThread.java:5571) 
        at java.lang.reflect.Method.invoke(Native Method) 
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:745) 
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:635) 
```

**Screenshots of the issue**

![20180524_122559](https://user-images.githubusercontent.com/26416942/40469628-cf6c92c6-5f4e-11e8-9cc5-62e09b928125.gif)


**Expected Bounty**

**20**

--- a/app/src/main/java/com/codingblocks/chatter/NoNetworkActivity.java
+++ b/app/src/main/java/com/codingblocks/chatter/NoNetworkActivity.java
@@ -12,6 +12,7 @@ import android.view.WindowManager;
 import android.widget.Button;
 
 import butterknife.BindView;
+import butterknife.ButterKnife;
 
 public class NoNetworkActivity extends AppCompatActivity {
 
@@ -28,6 +29,7 @@ public class NoNetworkActivity extends AppCompatActivity {
         );
 
         setContentView(R.layout.no_network_activity);
+        ButterKnife.bind(this);
 
         final String calledFrom = getIntent().getStringExtra("calledFrom");
 

